tests/overload.erl

```diff
@@ -127,15 +117,10 @@ test_vnode_protection(Nodes, BKV, ConsistentType) ->
 
     lager:info("Suspending vnode proxy for ~p", [Victim]),
     Pid = suspend_vnode_proxy(Victim),
-    ProcFun2 = fun(X) ->
-                      lager:info("in test_vnode_protection after suspend ProcFun, Procs:~p, Metric:~p", [X, ?NUM_REQUESTS]),
-                      X >= ?NUM_REQUESTS
-
-              end,
-    QueueFun2 = fun(X) ->
-                      lager:info("in test_vnode_protection after suspend QueueFun, QueueSize:~p, Metric:~p", [X, (?THRESHOLD * 1.1)]),
-                      X =< (?THRESHOLD * 1.1)
-              end,
+    ProcFun2 = build_predicate_gte("test_vnode_protection after suspend",
```

Comment:
As far as I know we can't, but since it's gte it's basically just defensive coding. Or at least that's how I read the original, which made enough sense to keep I think